OSDOCS-4468: updating MicroShift intro (#52594)
openshift/openshift-docs: microshift_getting-started/microshift-architecture.adoc

:_content-type: ASSEMBLY
[id="microshift-architecture"]
= Architecture
include::_attributes/attributes-microshift.adoc[]
:context: microshift-architecture

toc::[]

> Determine which version of {product-title} you have, find out about known issues, and troubleshoot common problems.

Reviewer: This looks like the same intro that appears in the troubleshooting page. Is that what we want?

Author: Nope, that's a mistake, good catch.

:_content-type: ASSEMBLY
[id="microshift-architecture"]
= Architecture
include::_attributes/attributes-microshift.adoc[]
:context: microshift-architecture

toc::[]

== The benefits of running containers on the edge

{product-title} is a single-node container orchestration runtime designed for running applications in low-resource edge environments. Like {OCP}, {product-title} is based on Kubernetes and retains many of the benefits of using containers, bringing those benefits to the edge. Because edge and small-form-factor computing are by nature created with only the essentials needed to operate, so is {product-title}.

== Key differences from {OCP}
* Works offline and in low-connectivity environments
* You do not need to perform external orchestration

> * You can use several single-node deployments in locations where full control is needed

Author: Yes, I made one last little nit on it. I'll put it out for Docs reviews next.

:_content-type: ASSEMBLY
[id="microshift-architecture"]
= Architecture
include::_attributes/attributes-microshift.adoc[]
:context: microshift-architecture

toc::[]

> 

Reviewer 2: It seems like there should be an abstract here between the "Architecture" heading and the "benefits" heading.

Author: Reworked it a bit. LMK what you think.

== The benefits of running containers on the edge

> {product-title} is a single-node container orchestration runtime designed for running applications in low-resource edge environments. Like {OCP}, {product-title} is based on Kubernetes and retains many of the benefits of using containers, bringing those benefits to the edge. Because edge and small-form-factor computing are by nature created with only the essentials needed to operate, so is {product-title}.

Reviewer 2: ```suggestion
{product-title} is a single-node container orchestration runtime designed for running applications in low-resource edge environments. Like {OCP}, {product-title} is based on Kubernetes and retains many of the benefits of using containers, bringing those benefits to the edge. {product-title} contains only the essentials needed to operate in edge and small form factor computing environments.
```
Rewording suggestion to avoid passive voice.

== Key differences from {OCP}

> * Works offline and in low-connectivity environments

Reviewer 2: This could use some clarification. We have [documentation](https://docs.openshift.com/container-platform/4.11/installing/installing_vsphere/installing-restricted-networks-installer-provisioned-vsphere.html) for installing in offline/highly restricted environments for OCP.

> * You do not need to perform external orchestration

Reviewer 2: Can you be more specific about what external orchestration means here? Is it referring to things like the advanced cluster management tool or something else? Like my similar comment, there are cases in OCP where clusters are set up in disconnected environments that don't have any external access for management/orchestration

Author: Yes, it isn't managed after install and deployment and there are no consoles, etc. I'll comment these two out for today and work on these bullets in a later PR.

> * You can use in locations where full control is needed

Reviewer 2: This could be more specific about what is meant by "full control" and how MicroShift helps with those scenarios. Without more detail, this applies to OCP as well

> * You can use several single-node deployments to retain the availability benefits of Kubernetes

Author: Something maybe about keeping core functions separate instead?

> * You can use several single-node deployments to retain the availability benefits of Kubernetes

Reviewer: MicroShift uses fewer resources, but also supports fewer APIs, than SNO OCP. The internal FAQ has some good language, maybe we can reuse some of that? https://docs.google.com/document/d/1vPR2seEZfs_lFRdh07IlcBVgcqYOfAWTBlh1yU-KkFQ/edit#heading=h.8fg6ro1kvi2q

> * You can use several single-node deployments to retain the availability benefits of Kubernetes

Author: Maybe the "full control" concept is good here. Goes along with the "plug n play" style features.

> * You can use several single-node deployments to retain the availability benefits of Kubernetes

Reviewer: Yes, I think if this section covers the resources/features trade-off and mentions the control aspect that would be good.
